src/sentry/event_manager.py

```diff
@@ -510,7 +511,23 @@ def save_error_events(
         try:
             group_info = assign_event_to_group(event=job["event"], job=job, metric_tags=metric_tags)
 
-        except HashDiscarded:
+        except HashDiscarded as e:
+            tombstone_id = getattr(e, "tombstone_id", None)
+            if tombstone_id:
+                try:
+                    group_tombstone = GroupTombstone.objects.get(id=tombstone_id)
+                    group_tombstone.last_seen = max(
+                        job["event"].datetime, group_tombstone.last_seen
+                    )
+                    group_tombstone.times_seen += 1
```

Comment:
Could there be a high volume group here that is discarded? If so, we could end up hammering postgres for each event that hits that group. Usually we use buffers for this kind of update around groups

Reply:
I wasn't aware of the buffer - thanks for the feedback, it makes perfect sense. I've incorporated the feedback in the PR: https://github.com/getsentry/sentry/pull/93685